server_scripts/glassfish_administer_service.py

```diff
@@ -297,8 +296,9 @@ def _run_remote_command(self, *cmd):
 
 if __name__ == '__main__':
     args = _parseArgs()
+    _VERBOSE = args.verbose
```

Comment:
Don't make this a global variable, you philistine. Pass it into the CommandGlassfishDomain class and make it an instance variable.